scripts/versioning/create_github_release.rb

```diff
@@ -1,22 +1,14 @@
-require 'net/http'
+require 'faraday'
 require 'json'
-require 'uri'
 
-begin
-  args = ARGV
-  if args.size != 6
-   raise ArgumentError, "Invalid usage: use `ruby create_github_release.rb {{repo_owner}} {{repo_name}} {{access_token}} {{tag_name}} {{release_name}} {{release_notes}}`"
-  end
-  repo_owner = args[0]
-  repo_name = args[1]
-  access_token = args[2]
-  tag_name = args[3]
-  release_name = args[4]
-  release_notes = args[5]
-
-  puts release_notes
+def create_github_release(repo_owner, repo_name, access_token, tag_name, release_name, release_notes, file_path=nil)
   github_api_url = "https://api.github.com/repos/#{repo_owner}/#{repo_name}/releases"
 
+  headers = {
+    'Authorization' => "token #{access_token}",
+    'Accept' => 'application/vnd.github.v3+json'
+  }
```

Comment:
should this contain `'Content-Type' => 'application/json'` to not merge it here https://github.com/twilio/twilio-verify-passkeys/pull/26/files#diff-f8c27b14886516cdfe45818912c865bfe63b7bf1c27eed49918af3f844895dcbR21?

Reply:
Done, Thank you 👍